File: app/src/main/java/com/example/asiantech/playmusic/model/Song.java
package com.example.asiantech.playmusic.model;

import java.io.Serializable;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * @author hoaht
 */
@Data
@AllArgsConstructor(suppressConstructorProperties = true)
public class Song implements Serializable {
    private long id;
    private String title;
    private long artistId;
    private String artist;
    private long albumId;
    private String album;
    private String display;
    private boolean isPlaying;
}

File: app/src/main/java/com/example/asiantech/playmusic/model/Song.kt
package com.example.asiantech.playmusic.model

import java.io.Serializable

/**
 * @author hoaht
 */
class Song(var id: Long, var title: String, var artistId: Long, var artist: String,
           var albumId: Long, var album: String, var display: String, var isPlaying: Boolean) : Serializable
